site/production-checklist.xml

```diff
@@ -0,0 +1,290 @@
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:doc="http://www.rabbitmq.com/namespaces/ad-hoc/doc"
+      xmlns:xi="http://www.w3.org/2003/XInclude"
+      xmlns:x="http://www.rabbitmq.com/2011/extensions">
+  <head>
+    <title>Production Checklist</title>
+  </head>
+  <body>
+    <doc:section name="intro">
+      <doc:heading>Introduction</doc:heading>
+
+      <p>
+        Data services such as RabbitMQ often have many tunable
+        parameters. Some configurations make a lot of sense for
+        development but are not really suitable for production.  No
+        single configuration fits every use case.  It is, therefore,
+        important to assess your configuration before going into
+        production. This guide aims to help with that.
+      </p>
+    </doc:section>
+
+
+    <doc:section name="users-and-permissions">
+      <doc:heading>Virtual Hosts, Users, Permissions</doc:heading>
+
+      <doc:subsection name="virtual-hosts">
+        <doc:heading>Virtual Hosts</doc:heading>
+        <p>
+          In a single-tenant environment, for example, when your RabbitMQ
+          cluster is dedicated to power a single system in production,
+          using default virtual host (<code>/</code>) is perfectly fine.
+        </p>
+
+        <p>
+          In multi-tenant environments, use a separate vhost for
+          each tenant/environment, e.g. <code>project1_development</code>,
+          <code>project1_production</code>, <code>project2_development</code>,
+          <code>project2_production</code>, and so on.
+        </p>
+      </doc:subsection>
+
+      <doc:subsection name="users">
+        <doc:heading>Users</doc:heading>
+        <p>
+          For production environments, delete the default user (<code>guest</code>).
+          Default user only can connect from localhost by default, because it has well-known
+          credentials. Instead of enabling remote connections, consider using a separate user
+          with administrative permissions and a generated password.
+        </p>
+        <p>
+          It is recommended to use a separate user per application. For example, if you
+          have a mobile app, a Web app, and a data aggregation system, you'd have 3
+          separate users. This makes a number of things easier:
+
+          <ul>
+            <li>Correlating client connections with applications</li>
+            <li>Using <a href="/access-control.html">fine-grained permissions</a></li>
+            <li>Credentials roll-over in case of a breach or periodically</li>
```

Comment:
Then perhaps change the order, since "in case of a breach" should be the less common occurring event. A matter of preference anyway…